ios/MapAnnotation.h

```diff
@@ -0,0 +1,18 @@
+//
+//  MapAnnotation.h
+//  TestAnnotationMoving
+//
+//  Created by Nishith Shah on 14/11/16.
```

Comment:
Please replace commented lines with following. -
/**
 * Appcelerator Titanium Mobile
 * Copyright (c) 2009-Present by Appcelerator, Inc. All Rights Reserved.
 * Licensed under the terms of the Apache Public License
 * Please see the LICENSE included with this distribution for details.
 */